packages/core/components/DataTable/DataTable.tsx

```diff
@@ -125,6 +125,16 @@ const DataTable = (props: DataTableProps) => {
     maxHeight: config.table.limitHeight && `${config.table.height}px`,
     OverflowY: 'scroll'
   }
+  const [height, getTableHeight] = useState(0)
+  const tableHgt = useRef(null)
```

Comment:
Best to spell out `tableHeight`. Its a few extra characters but will avoid confusion when other developers work on the app later on.